rollup.config.js

```diff
@@ -0,0 +1,89 @@
+import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
+import autoExternal from 'rollup-plugin-auto-external';
+import copy from 'rollup-plugin-copy';
+import replace from 'rollup-plugin-replace';
+import { terser } from 'rollup-plugin-terser';
+import resolve from 'rollup-plugin-node-resolve';
```

Comment:
No reason in node resolve plugin as you do not have any dependency (except peer ones).